package.json

```diff
@@ -12,7 +12,7 @@
         "name": "Weida Hong",
         "url": "https://blog.bitisle.net"
     },
-    "version": "0.6.2",
+    "version": "0.7.0-alpha",
```

Comment:
This change is used to indicated that the local installed version is debugging one.